bin/i18nlint

```diff
@@ -1 +0,0 @@
-../server/i18nlint/bin/i18nlint-cli.js
```

Comment:
I suspect this is a perms change without actual content change. I will try to extricate this meaningless change from the commit if desired.